specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -1960,6 +2312,119 @@
         "id"
       ],
       "description": "Virtual Network rule."
+    },
+    "Snapshot": {
+      "type": "object",
+      "description": "Response for Volume Snapshot request.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of Volume Snapshot.",
+          "$ref": "#/definitions/SnapshotProperties"
+        },
+        "systemData": {
+          "description": "Resource metadata required by ARM RPC",
+          "readOnly": true,
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
```

Comment:
Use the Resource definition from common types and remove this. Same comment for SnapshotCreateParameter.

Reply:
The resource definition differ from definition mentioned in our swagger. The tags field is mentioned under tracked resource in common types while we have it in Resource model.

Reply:
I don't think that is a very significant difference. And declaring Tags in 'Resource' is technically not really correct because only tracked resources always support Tags.

Reply:
@tadelesh From an sdk point of view, what do you think they should do?

Reply:
So @mentat9 is mentioning that Snapshot never should have had 'Tags' anyway, if it provides that functionality, it should be called something else. Has it already shipped with them though? Since it used to inherit them through the (incorrect) Resource definition?

Reply:
@rkmanda @mentat9 Any further thoughts on whether the 'no tags on proxy resources' rule merits any breaking changes?